components/team_selector_modal/team_selector_modal.tsx

```diff
@@ -130,19 +147,19 @@ export default class TeamSelectorModal extends React.PureComponent {
         }
     }
 
-    handleDelete = (values) => {
+    handleDelete = (values: Array<any>) => {
         this.setState({values});
     }
 
-    search = (term, multiselectComponent) => {
+    search = (term: string, multiselectComponent: { state: { page: number; }; setState: (arg0: { page: number; }) => void; }) => {
         if (multiselectComponent.state.page !== 0) {
             multiselectComponent.setState({page: 0});
         }
         this.props.actions.setModalSearchTerm(term);
     }
 
-    renderOption = (option, isSelected, onAdd, onMouseMove) => {
-        var rowSelected = '';
+    renderOption = (option: any, isSelected: boolean, onAdd: (value: any) => void, onMouseMove: (value: any) => void) => {
```

Comment:
option can be `Team` instead of `any`? If so, then the parameter to  `onAdd` and onMouseMove can be updated too?

Reply:
done in latest commit